plugins/postcss-is-pseudo/README.md

```diff
@@ -0,0 +1,234 @@
+# PostCSS Is Pseudo [<img src="https://postcss.github.io/postcss/logo.svg" alt="PostCSS" width="90" height="90" align="right">][postcss]
+
+[![NPM Version][npm-img]][npm-url]
+[![CSS Standard Status][css-img]][css-url]
+[![Support Chat][git-img]][git-url]
+
+[PostCSS Is Pseudo] lets you use the `:is` pseudo class function, following the
+[CSS Selector] specification.
+
+```css
+:is(input, button):is(:hover, :focus) {
+	order: 1;
+}
+```
+
+Becomes :
+
+```css
+input:hover {
+	order: 1;
+}
+input:focus {
+	order: 1;
+}
+button:hover {
+	order: 1;
+}
+button:focus {
+	order: 1;
+}
+```
+
+## Usage
+
+Add [PostCSS Is Pseudo] to your project:
+
+```bash
+npm install @csstools/postcss-is-pseudo --save-dev
+```
+
+Use [PostCSS Is Pseudo] as a [PostCSS] plugin:
+
+```js
+import postcss from 'postcss';
+import postcssIsPseudo from '@csstools/postcss-is-pseudo';
+
+postcss([
+  postcssIsPseudo(/* pluginOptions */)
+]).process(YOUR_CSS /*, processOptions */);
+```
+
+[PostCSS Is Pseudo] runs in all Node environments, with special instructions for:
+
+| [Node](INSTALL.md#node) | [Webpack](INSTALL.md#webpack) | [Create React App](INSTALL.md#create-react-app) | [Gulp](INSTALL.md#gulp) | [Grunt](INSTALL.md#grunt) |
+| --- | --- | --- | --- | --- |
+
+## Options
+
+### preserve
+
+The `preserve` option determines whether the original notation
+is preserved. By default, it is not preserved.
+
+```js
+postcss([
+  postcssIsPseudo({ preserve: true })
+]).process(YOUR_CSS /*, processOptions */);
+```
+
+```css
+:is(input, button):is(:hover, :focus) {
+	order: 1;
+}
+```
+
+Becomes :
+
+```css
+input:hover {
+	order: 1;
+}
+input:focus {
+	order: 1;
+}
+button:hover {
+	order: 1;
+}
+button:focus {
+	order: 1;
+}
+:is(input, button):is(:hover, :focus) {
+	order: 1;
+}
+```
+
+### doesNotExistName
+
+The `doesNotExistName` option allows you to change to selector used for `:not()` selectors.
```

Comment:
Let's add a note here that hints that this is needed and will be explained below (and potentially link to the shortcomings)

Reply:
Linked to the specificity docs and renamed to option to `specificityMatchingName`
It is still not fully clear from the name alone but better than `doesNotExistName`